sdk/storage/azblob/blob/client_test.go

```diff
@@ -191,67 +191,6 @@ func waitForCopy(_require *require.Assertions, copyBlobClient *blockblob.Client,
 	}
 }
 
-func (s *BlobUnrecordedTestsSuite) TestCopyBlockBlobFromUrlSourceContentMD5() {
```

Comment:
is this a no longer supported scenario?

Reply:
This test is moved to blockblob/client_test.go.